app/javascript/src/components/Dashboard/Notes/NotesList.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "neetoicons";
+import { Typography, Tag, Avatar, Dropdown } from "neetoui/v2";
+
+const NotesList = ({ notes, setShowDeleteAlert }) => {
```

Comment:
You can name the file `List.jsx`. Since it resides inside of `Notes`, we don't explicitly need to prefix it with `Notes`.